development/run-version-bump.js

```diff
@@ -0,0 +1,44 @@
+const { promisify } = require('util')
+const fs = require('fs')
+const readFile = promisify(fs.readFile)
+const writeFile = promisify(fs.writeFile)
+const path = require('path')
+const changelogPath = path.join(__dirname, '..', 'CHANGELOG.md')
+const manifestPath = path.join(__dirname, '..', 'app', 'manifest.json')
+const manifest = require('../app/manifest.json')
+const versionBump = require('./version-bump')
+
+const bumpType = normalizeType(process.argv[2])
+
+
+readFile(changelogPath)
```

Comment:
to deal with top-level async i usually do
```js
start().then(console.log).catch(console.error)

async function start(){
// ...
}
```

then put everything inside `start`